examples/text_classification/ios/TextClassification.xcodeproj/project.pbxproj

```diff
@@ -387,18 +387,17 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3145779E2698069303666999 /* Pods-TextClassification.debug.xcconfig */;
 			buildSettings = {
-				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = H83UK2M7VU;
```

Comment:
You probably don't want to leave your development team code in all of these PRs, but you can remove those in a separate PR since these will all be merged in to the main repo tomorrow.